dashboard/modules/dashboard_sdk.py

```diff
@@ -135,8 +135,19 @@ def parse_cluster_info(
 ) -> ClusterInfo:
     module_string, inner_address = _split_address(address)
 
-    # If user passes http(s):// or ray://, go through normal parsing.
-    if module_string in {"http", "https", "ray"}:
+    # If user passes in ray://, raise error. Cluster-level submission should
```

Comment:
What's cluster-level submission?

Reply:
Or what's the alternative? (job level submission?)

Reply:
yeah I'm not sure, the two options are ray client and dashboard

Reply:
I'll use dashboard.